src/widget/form/settings/aboutform.cpp

```diff
@@ -46,6 +46,8 @@ void AboutForm::replaceVersions()
                                                        QString::number(TOX_VERSION_MAJOR) + "." +
                                                        QString::number(TOX_VERSION_MINOR) + "." +
                                                        QString::number(TOX_VERSION_PATCH)));
+    bodyUI->qtVersion->setText(
+                bodyUI->qtVersion->text().replace("$QTVERSION", QT_VERSION_STR));
```

Comment:
Tried to avoid %1 in designer mode, so it's more obvious when editing XML in UI.